.github/workflows/ci.yml

```diff
@@ -59,6 +59,16 @@ jobs:
           python -c 'from deluge._libtorrent import lt; print(lt.__version__)';
           catchsegv python -X dev -m pytest -v -m "not (todo or gtkui or security)" deluge
 
+      - name: Test security
+        # if: "contains(github.event.commits[0].message, 'security_test')"
+        run: |
+          SECURITY_TESTS=True
```

Comment:
This doesn't actually work since it's not exported for pytest. To use `env` job keyword to set env var in an action: https://docs.github.com/en/actions/learn-github-actions/environment-variables

Thinking further maybe instead remove the `or security` from pytest step and add a step before `install dependencies` that sets the env vars, something like this:

```
      - name: Sets env var for security
        if: contains(github.event.head_commit.message, 'security_test')
        run: echo "SECURITY_TESTS=True" >> $GITHUB_ENV

      - name: Install dependencies
      ...

      - name: Install security dependencies
        if contains(env.SECURITY_TESTS, 'True')
        run: |
          wget -O- $TESTSSL_URL | tar xz
          mv -t deluge/tests/data testssl.sh-$TESTSSL_VER/testssl.sh testssl.sh-$TESTSSL_VER/etc/;
        env:
          TESTSSL_VER=3.0.1
          TESTSSL_URL=https://codeload.github.com/drwetter/testssl.sh/tar.gz/refs/tags/$TESTSSL_VER
```

If you use the latest version 3.0.6 then the TESTSSL_URL needs to a `v` prefixed e.g. `.../tags/v$TESTSSL_VER`

Reply:
sounds good. changed to this implementation.